data/mods/Xedra_Evolved/mutations/werewolf_traits.json

```diff
@@ -0,0 +1,93 @@
+[
+  {
+    "//": "Werewolf traits should focus on hunting, fighting extradimensional threats, the moon, battle fury, shapeshifting",
+    "type": "mutation",
+    "id": "WEREWOLF_MANA_TRAIT",
+    "name": { "str": "Werewolf Mana", "//~": "NO_I18N" },
+    "points": 0,
+    "description": { "str": "Your mana is calculated differently because you're a werewolf.", "//~": "NO_I18N" },
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "player_display": false,
+    "enchantments": [
+      {
+        "values": [ { "value": "MAX_MANA", "add": { "math": [ "(-1000 * (1 + ( (0.2 + (u_val('intelligence') * 0.1 ) - 1) ) )" ] } } ]
+      }
+    ]
+  },
+  {
+    "//": "Tier 1 traits begin here",
+    "type": "mutation",
+    "id": "WEREWOLF_SMELL_NEARBY_ENEMIES",
+    "name": { "str": "Scent the Prey" },
+    "points": 2,
+    "description": "Your sense of smell is heightened even in human form, allowing you to smell nearby enemies. In war or wolf form, this range is extended.",
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "//2": "No one but the player has scents so done as a simple detect creatures trait with a very low range. This DOES allow you smell robots, but hopefully if we get JSONized detection we can limit the range of creatures",
+    "enchantments": [
+      {
+        "values": [
+          {
+            "value": "MOTION_VISION_RANGE",
+            "add": { "math": [ "2 + (u_has_trait('WEREWOLF_HYBRID_FORM_TRAITS') * 2) + (u_has_trait('VAMPIRE_WOLF_FORM_TRAITS') * 4)" ] }

```

Comment:
I'm having trouble tracking `VAMPIRE_WOLF_FORM_TRAITS` this down?

Reply:
It's here: https://github.com/CleverRaven/Cataclysm-DDA/blob/91af05bfb461a24fb9869603db5c1348aa9ad98f/data/mods/Xedra_Evolved/mutations/mutations.json#L680

I've just been reusing the vampire version for other people who can turn into a wolf (pooka and werewolves). I probably should have given it a more generic id but it's too late now